### PR TITLE
[EPO-2968] Update DistanceCalculator to store updated data structure.

### PR DESCRIPTION
**Pull request by georgegomez144**
Store DistanceCalculator data/content in the same way the
SizeCalculator does.

### CODE REVIEW

File: src/data/pages/distance-to-supernovae-2.json [hunk 1]
```diff
@@ -53,7 +49,7 @@
           "label": "<p>Enter the peak apparent magnitude (m) for supernova #4 to find its distance from Earth.</p>",
           "placeholder": "Enter peak magnitude",
           "answerPre": "<p>Selected: </p>",
-          "answerAccessor": "text"
+          "answerAccessor": "magnitude"
```
Comment (blnkt) [inline src/data/pages/distance-to-supernovae-2.json:52]: I'm not sure what this change is accomplishing.  If anything wouldn't this change to like parsecs or megaLightyears or distance or something?  Or even just null?  In any case you're never using the answerAccessor in the DistanceCalculator and if some other component was using a DistanceCalculator answer for something we wouldn't want it to grab the magnitude off the answer data we'd want it to grab one of the distance measurements right?

File: src/data/pages/distance-to-supernovae-2.json [hunk 2]
```diff
@@ -65,7 +61,7 @@
           "title": "Question #1",
           "label": "<p>Enter the peak apparent magnitude (m) for supernova #5 to find its distance from Earth.</p>",
           "answerPre": "<p>Selected: </p>",
-          "answerAccessor": "text"
+          "answerAccessor": "magnitude"
```
Comment (blnkt) [inline src/data/pages/distance-to-supernovae-2.json:64]: same as above

File: src/data/pages/distance-to-supernovae-2.json [hunk 3]
```diff
@@ -77,7 +73,7 @@
           "title": "Question #1",
           "label": "<p>Enter the peak apparent magnitude (m) for supernova #6 to find its distance from Earth.</p>",
           "answerPre": "<p>Selected: </p>",
-          "answerAccessor": "text"
+          "answerAccessor": "magnitude"
```
Comment (blnkt) [inline src/data/pages/distance-to-supernovae-2.json:76]: same as above

File: src/components/qas/questions/qaCalculators/distanceCalculator/index.jsx [hunk 4]
```diff
@@ -69,69 +91,67 @@ class DistanceCalculator extends React.PureComponent {
     }
   }
 
-  updateCalculatedMeasurements(value) {
-    const peakMagnitude = +value !== 0 ? +value : 'm';
+  getCalculatedMeasurements(value) {
+    const { value: oldValue } = this.state;
+
+    if (!value) return oldValue;
```
[the remark this answers is not in the document]
Comment (georgegomez144) [reply src/components/qas/questions/qaCalculators/distanceCalculator/index.jsx:97]: So, I'm just going to remove the superfluous check in the getCalculatedMeasurements.

**Comment by blnkt** [comment]
closing because will be delivered in #134

**Pull request closed without merging**